server/src/main/java/org/candlepin/dto/api/v1/InfoAdapter.java

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright (c) 2009 - 2020 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.dto.api.v1;
```

Comment:
I would not put this in this package, since it isn't part of our API. It's a utility class for internal transformations of data. I'd say `org.candlepin.resource.util` is more appropriate

Reply:
Util is probably best if we're going to have a separate class for it. However, we'll need to be careful and ensure that any place which uses the functions are heavy scrutinized so we don't end up using this in places that should be updated in a different way.

Reply:
Class relocated to `org.candlepin.resource.util` package.